### ISSUE TITLE
How to automate testing words in Foma?

### ISSUE THREAD

**Issue by puputrizqiyah** [mhulden/foma#135]
Hello, i would like to ask how to automate testing words in foma, so far i know foma could only test word by word, meanwhile i have around 600 test cases that would be tested on Foma and it is impossible to type it one by one. Can i test those 600 words in the same time without type it one by one?

**Comment by AmbientLighter** [comment]
Yes, you can put them all in foma script. Alternatively, you may call it from Python script. There are some examples in documentation.
